apps/studio/components/ui/AIAssistantPanel/AIOnboarding.tsx

```diff
@@ -33,51 +32,42 @@ export const AIOnboarding = ({
       title: 'Create a back-end',
```

Comment:
Take the `defaultPrompts` out of the component, they're causing rerenders because a new array is created each render.